content/kubernetes/active-active/preview/create-reaadb.md

```diff
@@ -103,8 +103,8 @@ Before creating an Active-Active database on Redis Enterprise for Kubernetes, yo
       name: example-aadb-1
     spec:
```

Comment:
imo we should also include the '.spec.databaseSecretName' that sets the AA database password, we are thinking that this should be recommendation (currently this example will create AA DB without password and it is not a good practice), maybe adding a note that this is example w/o the database secret and directing to "how to set database secret" instructions - wdyt?.